sql/core/src/main/scala/org/apache/spark/sql/catalyst/analysis/InvokeProcedures.scala

```diff
@@ -36,20 +37,35 @@ class InvokeProcedures(session: SparkSession) extends Rule[LogicalPlan] {
     case c: Call if c.resolved && c.bound && c.execute && c.checkArgTypes().isSuccess =>
       session.sessionState.optimizer.execute(c) match {
         case Call(ResolvedProcedure(_, _, procedure: BoundProcedure), args, _) =>
-          invoke(procedure, args)
+          invoke(procedure, args, c)
         case _ =>
           throw SparkException.internalError("Unexpected plan for optimized CALL statement")
       }
   }
 
-  private def invoke(procedure: BoundProcedure, args: Seq[Expression]): LogicalPlan = {
+  private def invoke(procedure: BoundProcedure, args: Seq[Expression],
+      callCommand: Call): LogicalPlan = {
     val input = toInternalRow(args)
     val scanIterator = procedure.call(input)
     val relations = scanIterator.asScala.map(toRelation).toSeq
+    // Wrap the result in a CommandResult so SparkConnect
```

Comment:
It seems reasonable to switch to `CommandResult` for now. We will have to adapt `SparkConnectPlanner` eventually to support `MultiResult`. No need to look into that now.